src/java/org/apache/cassandra/db/commitlog/CommitLog.java

```diff
@@ -177,7 +182,7 @@ public void recoverPath(String path) throws IOException
 
     private static UUID getLocalHostId()
     {
-        return Optional.ofNullable(StorageService.instance.getLocalHostUUID()).orElseGet(SystemKeyspace::getLocalHostId);
+        return StorageService.instance.getLocalHostUUID();
```

Comment:
if I understand that correctly, this will never be null anymore because we are doing that initialization in `CassandraDaemon.`